R/air.R

```diff
@@ -0,0 +1,213 @@
+#' Configure a project to use Air
+#'
+#' @description
+#' [Air](https://github.com/posit-dev/air) is an extremely fast R code
```

Comment:
Same comment as for NEWS re: which URL is best.

Reply:
I'm going to leave this one for now because its possible the main website url could get redirected to something more official than github pages in the near-ish term, and I dont want usethis to have to deal with redirect issues.